lib/src/transformations/units_of_measurement_extensions.dart

```diff
@@ -831,300 +831,3 @@ extension UnitsOfMeasurementDecimalExtensions on Decimal {
   /// Gets a [Weight] representing this number of gigatonnes.
   Weight gigatonnes() => Weight.fromGigatonnes(this);
 }
-
-/// Provides humanizing extensions on [Area].
```

Comment:
All these extensions are no longer needed. It was a brain fart to add them in the first place. UoMs are types defined by humanizer, not by some third party, so extensions for humanizing them don't make a lot of sense. Moreover, instead of having a separate `humanize` method on those types, the `toString` can be leveraged to provide a default humanization of that UoM.